lib/src/components/NxFilterDropdown/NxFilterDropdown.tsx

```diff
@@ -68,12 +68,16 @@ function NxFilterDropdownRender<T extends string | number = string>(props: Props
 
   function onResetClick() {
     onChange(new Set());
+    if (menuRef.current) {
+      menuRef.current.focus();
```

Comment:
As I mentioned in another comment, I'm not seeing anything indicating that focusing the menu itself is truly a part of the pattern.  If we get rid of that, what you could do here instead is focus the dropdown toggle.